app/src/main/java/com/ex/org/exceptionstest/com/ex/lib/ExceptionReporter.java

```diff
@@ -48,11 +48,12 @@ public static ExceptionReporter getInstance(Application app){
 
 
     public void init(){
+        //ERIK - it might be nice to prevent this from being called twice
         interceptExceptions();
     }
 
     private void interceptExceptions() {
-        Thread.UncaughtExceptionHandler defaultExceptionHandler = Thread.getDefaultUncaughtExceptionHandler();
+        final Thread.UncaughtExceptionHandler defaultExceptionHandler = Thread.getDefaultUncaughtExceptionHandler();
```

Comment:
why does it have to be final?

Reply:
In Android Studio 3.0, Google does some magic to allow you to use Java 8 language features, like automatically making stuff static when referencing them from inside an anonymous inner class. To make it build in AS 2.3 I dropped Java 8 support (you could still get that using RetroLambda if needed)

Reply:
got it.. yeah.. it'll be a lib so it's better to drop support for it. cheers